Warsaw_Tram/AlertHelper.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  AlertHelper.swift
+//  Warsaw_Tram
+//
+//  Created by Małgorzata Dziubich on 25/06/16.
+//  Copyright © 2016 Małgorzata Dziubich. All rights reserved.
+//
+
+import UIKit
+
+protocol AlertHelperProtocol {
+    func showAlert(title: String, message: String, okAction: UIAlertAction, cancelAction: UIAlertAction, presentingViewController: UIViewController)
+    func showAlert(title: String, message: String, okAction: UIAlertAction, presentingViewController: UIViewController)
+}
+
+class AlertHelper: NSObject, AlertHelperProtocol {
```

Comment:
I'd rather extend `AlertHelperProtocol` with default implementation for `UIViewController`